test/Connector.t.sol

```diff
@@ -180,71 +182,290 @@ contract ConnectorTest is Test {
+        string memory domainName = "Centrifuge";
+        uint32 domainId = 3000;
+        bytes32 recipient = stringToBytes32("0xefc56627233b02ea95bae7e19f648d7dcd5bb132");
+        user = address(this); // set deployer as user to approve the cnnector to transfer funds
 
-        // TODO: fix assertions
+        // add Centrifuge domain to router                  
+        bridgedRouter.enrollRemoteRouter(domainId, recipient);
+        
+        // add Centrifuge domain to connector
+        assertEq(bridgedConnector.wards(address(this)), 1);
+        bridgedConnector.file("domain", domainName, domainId);
+        bridgedConnector.deny(address(this)); // revoke ward permissions to test public functions
+    
+        // fund user
+        homeConnector.addPool(poolId);
+        homeConnector.addTranche(poolId, trancheId, "Some Name", "SYMBOL");
+        homeConnector.updateMember(poolId, trancheId, user, uint(-1));
+        homeConnector.transfer(poolId, trancheId, user, amount);
+        
+        // approve token
+        (address token_,,) = bridgedConnector.tranches(poolId, trancheId);
+        RestrictedTokenLike token = RestrictedTokenLike(token_);
+        token.approve(address(bridgedConnector), uint(-1)); // approve connector to take token
+
+        // transferTo
+        vm.expectRevert(bytes("CentrifugeConnector/domain-does-not-exist"));
+        bridgedConnector.transferTo(poolId, trancheId, user, amount, "Unknown"); // use unknown domain name
+     }
+    
+    function testTransferToUnknownDomainIDFails(uint64 poolId, bytes16 trancheId, uint256 amount, address user) public {
+        string memory domainName = "Centrifuge";
+        uint32 domainId = 3000;
+        bytes32 recipient = stringToBytes32("0xefc56627233b02ea95bae7e19f648d7dcd5bb132");
+        user = address(this); // set deployer as user to approve the cnnector to transfer funds
+
+        // add Centrifuge domain to router                  
+        bridgedRouter.enrollRemoteRouter(domainId, recipient);
+        
+        // add Centrifuge domain to connector
+        assertEq(bridgedConnector.wards(address(this)), 1);
+        bridgedConnector.file("domain", domainName, 5000); // use wrong domainID
+        bridgedConnector.deny(address(this)); // revoke ward permissions to test public functions
+    
+        // fund user
+        homeConnector.addPool(poolId);
+        homeConnector.addTranche(poolId, trancheId, "Some Name", "SYMBOL");
+        homeConnector.updateMember(poolId, trancheId, user, uint(-1));
+        homeConnector.transfer(poolId, trancheId, user, amount);
+        
+        // approve token
+        (address token_,,) = bridgedConnector.tranches(poolId, trancheId);
+        RestrictedTokenLike token = RestrictedTokenLike(token_);
+        token.approve(address(bridgedConnector), uint(-1)); // approve connector to take token
+
+        // transferTo
+        vm.expectRevert(bytes("!remote"));
```

Comment:
This is an odd revert message. Is there no way to make this more verbose?

Reply:
the Nomad Router contract that we are inheriting from is throwing that one